### PR TITLE
Documentation change to enhance readability

### PR DESCRIPTION
**Pull request by vijayasarathib**
Update custom-view-plugin.rst to increase readability

---
**^ Add meaningful description above**

Read the **[Pull Request Guidelines](https://github.com/apache/airflow/blob/main/CONTRIBUTING.rst#pull-request-guidelines)** for more information.
In case of fundamental code changes, an Airflow Improvement Proposal ([AIP](https://cwiki.apache.org/confluence/display/AIRFLOW/Airflow+Improvement+Proposals)) is needed.
In case of a new dependency, check compliance with the [ASF 3rd Party License Policy](https://www.apache.org/legal/resolved.html#category-x).
In case of backwards incompatible changes please leave a note in a newsfragment file, named `{pr_number}.significant.rst` or `{issue_number}.significant.rst`, in [newsfragments](https://github.com/apache/airflow/tree/main/newsfragments).

### CODE REVIEW

File: docs/apache-airflow/howto/custom-view-plugin.rst [hunk 1]
```diff
@@ -24,12 +24,12 @@ core UI using the Plugin manager
 
 This is an example plugin for Airflow that displays absolutely nothing.
 
-In this plugin, two object reference are derived from the base class
+In this plugin, two objects reference are derived from the base class
```
Comment (Lee-W) [inline docs/apache-airflow/howto/custom-view-plugin.rst]: Should it be `two object references` instead?

Comment (uranusjr) [reply docs/apache-airflow/howto/custom-view-plugin.rst]: ↑ Yes

File: docs/apache-airflow/howto/custom-view-plugin.rst [hunk 2]
```diff
@@ -24,12 +24,12 @@ core UI using the Plugin manager
...
 ``airflow.plugins_manager.AirflowPlugin``. They are flask_blueprints and
 appbuilder_views
 
 Using flask_blueprints in Airflow plugin, the core application can be extended
-to support the application that is customized to view Empty Plugin.
+to support the customized application to view the Empty Plugin.
```
Comment (uranusjr) [inline docs/apache-airflow/howto/custom-view-plugin.rst]: I think the _the_ is less right. Empty Plugin is a proper noun here and does not need a definite article.

File: docs/apache-airflow/howto/custom-view-plugin.rst (hunk 1 above)
Comment (vijayasarathib) [reply docs/apache-airflow/howto/custom-view-plugin.rst]: I have changed it now

File: docs/apache-airflow/howto/custom-view-plugin.rst (hunk 2 above)
Comment (vijayasarathib) [reply docs/apache-airflow/howto/custom-view-plugin.rst]: sure

**Review by vincbeck — APPROVED** [review]

**Review by amoghrajesh — APPROVED** [review]
Pretty straight forward PR. Looks good +1

**Review by potiuk — APPROVED** [review]

**Pull request merged**